src/main/java/com/crud/tasks/service/SimpleEmailService.java

```diff
@@ -33,6 +33,7 @@ private SimpleMailMessage createMailMessage(final Mail mail) {
         mailMessage.setTo(mail.getMailTo());
         mailMessage.setSubject(mail.getSubject());
         mailMessage.setText(mail.getMessage());
+        if (!"".equals(mail.getToCc()) && mail.getToCc() != null)  mailMessage.setCc(mail.getToCc());
```

Comment:
warunki powinny byc na odwrót. Najpierw musisz sprawdzic czy nie jest nullem, dopiero pozniej czy nie jest pustym Stringiem. Obecnie dla null'a poleci Ci NullPointerException

Reply:
uzyj gotowego api String'a : `!mail.getToCc().isEmpty()`

Reply:
po if'ie lepiej uzyc `{ }`. Java pozwala na ich pominiecie, ale to b. zla praktyka

Reply:
poprawione